<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductionVerify</class>
 <widget class="QWidget" name="ProductionVerify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>WenYue HouXianDaiTi (Non-Commercial Use)</family>
    <pointsize>28</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="spinBoxIntensity">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
border:1px solid rgb(29,165,219);
color:rgb(255,251,240);
background-color:rgba(0,0,0,0);
}

QSpinBox::up-button
{
  width:0px;
  height:0px;
}
QSpinBox::down-button
{
  width:0px;
  height:0px;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>司机侧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>副驾A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>副驾B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>左前门</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>右前门</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode3">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>左后门</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode4">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>110</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>右后门</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntensity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(29,165,219);
}</string>
   </property>
   <property name="text">
    <string>亮度:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderIntensity">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>1111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::add-page:Horizontal
{
  background-color: rgba(87, 97, 106,0);
  height:40px;
}
QSlider::sub-page:Horizontal
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29,165,219, 180), stop:1 rgba(29,165,219, 180));
  height:40px;
}
QSlider::groove:Horizontal
{
  background:transparent;
  height:46px;
}
QSlider::handle:Horizontal
{
  background-color:rgb(29,165,219);
  height: 40px;
  width:30px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>640</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(255,251,240);
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(29,165,219);
}</string>
   </property>
   <property name="text">
    <string>氛围灯简单控制盒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeColorManual">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>手动改变颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeColorAuto">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>460</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>开始自动改变颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode12">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>天窗右</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNode11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>631</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:2px solid #0fbacd;
border-top-right-radius: 15px;
border-bottom-left-radius: 15px;
color:rgb(0,255,255);
ont-size:22px;
background:rgba(29, 165, 219, 0.3);
}</string>
   </property>
   <property name="text">
    <string>天窗左</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>681</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/PICS/resources/logoNeicon.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
